src/App.js

```diff
@@ -7,21 +7,24 @@ import SectionsView from './components/admin/SectionsView';
 import {AdminHome, AdminLayout} from './components/admin/AdminLayout';
 import {StudentLayout, StudentHome} from './components/student/StudentLayout';
 import ScheduleView from './components/student/ScheduleView';
+import AddCourse from './components/student/CourseEnroll';
 import Transcript from './components/student/Transcript';
 import StudentAssignmentsView from './components/student/AssignmentsStudentView';
 import InstructorLayout from './components/instructor/InstructorLayout';
 import InstructorHome from './components/instructor/InstructorHome';
 import AssignmentsView from './components/instructor/AssignmentsView';
 import EnrollmentsView from './components/instructor/EnrollmentsView';
 import InstructorSectionsView from './components/instructor/InstructorSectionsView';
+import CourseEnroll from "./components/student/CourseEnroll";
 
 
 function App() {
 
   // change to INSTRUCTOR or STUDENT for testing.  
   // when login is implemented, the user type will come from the logged in user's ROLE.
 
-  const userType = 'INSTRUCTOR'; // change to INSTRUCTOR or STUDENT for testing.  
+  //const userType = 'INSTRUCTOR'; // change to INSTRUCTOR or STUDENT for testing.
+  const userType = 'STUDENT'; // change to INSTRUCTOR or STUDENT for testing.
```

Comment:
Maybe we just leave userType as 'STUDENT' for when we push? That way we don't keep changing it in git.